src/service/Configurator_6D.h

```diff
@@ -47,7 +47,7 @@ class Configurator_6D {
 	void init(CmdArgs const& args) noexcept;
 	void finalize() noexcept;
 private:
-
+	bool useModes=false;
```

Comment:
I think you don't need to introduce a member variable for that. it's good enough to declare it in the configurator init() function. I'll do that for you...

please get into the habit of including spaces. I think eclipse has a code formatter that automatically does that. We both should configure it to result into the same "code style". However, it's not a showstopper ;)